codegens/curl/lib/util.js

```diff
@@ -226,7 +226,8 @@ var self = module.exports = {
       .replace(/%5D/g, ']')
       .replace(/%7D/g, '}')
       .replace(/%25/g, '%')
-      .replace(/'/g, '%27');
+      .replace(/'/g, '%27')
+      .replace(/%2B/g, '+');
```

Comment:
@aman-v-singh Let's keep this at line no. 228. i.e. before we replace `%` chars.